src/views/Welcome.vue

```diff
@@ -63,7 +65,7 @@ export default {
       {icon: 'mdi-projector-screen-outline', title: 'Create a Project Site', path: '/create_project_site'},
       {icon: 'mdi-export', title: 'E-Grade Export', path: '/grade_export/:id'},
       {icon: 'mdi-account-school', title: 'Find a User to Add', path: '/add_user/:id'},
-      {icon: 'mdi-email-fast-outline', title: 'Mailing List', path: '/mailing_list/:id'},
+      {icon: 'mdi-email-fast-outline', title: 'Mailing List', path: '/mailing_list/1466'},
```

Comment:
@johncrossman is this hard-coded ID meant to be here?

Reply:
@lyttam - Good catch. :)  Yes, for now. That page won't render w/ a non-numeric. Ultimately, it needs to be a valid canvas-course-site id.

Reply:
Makes sense!